examples/tutorial-provision-container/appenv

```diff
@@ -20,7 +20,6 @@ import hashlib
 import http.client
```

Comment:
The file `batou` in this folder links to `../../appenv.py`, maybe this file should also link to the one canonical appenv version in this project.